CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - **decompress:** Fix nested Zstd archive extraction ([#4608](https://github.com/ScoopInstaller/Scoop/issues/4608))
 - **shim:** Fix PS1 shim error when in different drive in PS7 ([#4614](https://github.com/ScoopInstaller/Scoop/issues/4614))
 
+### Builds
```

Comment:
Is `Builds` the appropriate category here? I'm not so sure.

Reply:
I put all schema and bin changes under `Builds` since this has less CLI changes to end users.